Translation corrections (#402)
07th-mod/higurashi-console-arcs: Update/omot_tips_002.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>運送屋</color>", NULL, "<color=#f5e6d3>Deliveryman</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s20/00/unsob44001", 256, TRUE);
	OutputLine(NULL, "「え、そこにあった大型家具ですか？　それでしたら、今さっき遠嶋さんがトラックに積み込んで配送に出ちゃいましたよ」",
			NULL, "\"Oh, you mean that large piece of furniture? If that's what you're looking for, Toujima-san loaded it on the truck earlier and it's already out for delivery.\"", Line_Normal);
	ClearMessage();

//運送屋rvS20/00/unsoA44002.「ええ～！　ここにあったのって、明日配送の指定が入っていただろ？　今日持っていったら相手先に怒られるぞ！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f5e6d3>運送屋</color>", NULL, "<color=#f5e6d3>Deliveryman</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 0, "ps3/s20/00/unsoa44002", 256, TRUE);
	OutputLine(NULL, "「ええ～！　ここにあったのって、明日配送の指定が入っていただろ？　今日持っていったら相手先に怒られるぞ！」",

> 			NULL, "\"Huuh~?! But, wasn't it specifically scheduled for delivery tomorrow? If the customer receives it today they'll get mad at us!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Huuh~?! But, wasn't it specifically scheduled for delivery tomorrow? If the customer receives it today they'll be upset!\"", Line_Normal);
```
